osquery/tables/networking/windows/arp_cache.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <boost/range/algorithm/find.hpp>
+#include <osquery/tables.h>
+#include <string>
+
+#include "boost/algorithm/string/replace.hpp"
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+#include "osquery/tables/networking/windows/interfaces.h"
```

Comment:
Ah okay, the problem I have been finding is that the clang formatter tends to re-order these which can potentially break the linter. I usually don't mess with the order once I get it to compile but I'll give this encantation a shot :)